main/guides/ui-tutorial/conclusion/index.md

```diff
@@ -0,0 +1,20 @@
+## Conclusion
+
+Throughout this tutorial, you've accomplished all the basics of building an Agoric Dapp UI:
+
+- Scaffolding a new React App.
+- Setting up `AgoricProvider` and connecting to a wallet.
+- Reading and rendering purse balances.
+- Reading contract data with `chainStorageWatcher`.
```

Comment:
My thought here, and this also was a gotcha for me, but this tutorial is showing how you read data from vstorage in general, as the code is using

```
[Kind.Data, 'published.agoricNames.instance']
...
[Kind.Data, 'published.agoricNames.brand']
```

But these aren't storage paths set by the contract. I think the wording can improve here so a brand new dev doesn't think they are reading contract data.

I see an opportunity to even combine/link other resources on how the writing of vstorage works, so they can understand the relationship between how writes work and these storage paths they see in this tutorial.

For this specifically, we could even change this to be 

```
- Reading vstorage data with `chainStorageWatcher`
```

would love thoughts on this

Reply:
i also do understand the brand is from the contract, since its minting. Therefore the contract is still reading brand into from the asset being minted by the contract, but in a round-about way, this seems like we can say this more clear. Maybe its just we don't call it contract data here, but more vstorage data... idk 

So maybe, its just some clarity in the tutorial on how those storage paths can become where you store data if you want it to be assessible to the outside world.

Reply:
Yea that's understandable, I wasn't sure if "vstorage" would sound too unfamiliar but I don't think it's a big concern, just changed it from "contract data" to "vstorage" everywhere.

Also added a link to https://docs.agoric.com/guides/zoe/pub-to-storage.html#publishing-to-chainstorage as well